src/main/cpp/t06_longest.cpp

```diff
@@ -17,10 +17,26 @@
 
 #include "t06_longest.h"
 #include <iostream>
-
+#include <string>
 
 using namespace std;
 
 int t06_longest() {
-
+ string s;
+    int startMax = 0, endMax = 1, bufStart = 0, bufEnd = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'bufEnd' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719848)